src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+const  {check, param}  =  require ( 'express-validator' );
+const {StatusCodes} = require ('http-status-codes');
+const {validateResult} = require('../helpers/response-result');
+
+
+const validationAccount = [
+    check('lastname')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 2, max: 150}).withMessage('El apellido(s) deben tener un mínimo de 2 caracteres y un máximo de 150 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü ]+$/).withMessage('Solo se permiten letras y espacios'),
+    check('name')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 2, max: 150}).withMessage('El nombre(s) deben tener un mínimo de 2 caracteres y un máximo de 150 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü ]+$/).withMessage('Solo se permiten letras y espacios'),
+    check('age')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .not().isString().withMessage('El campo debe ser un número')
+        .custom((value, { request }) => {
+            if (value < 10 || value > 100) {
+                return false;
+            }
+            return true;
+        }).withMessage('Debe tener más de 10 años y menos de 100 años.'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        }).withMessage("Debe tener más de 10 años y menos de 100 años."),
```